tools/system_libs.py

```diff
@@ -872,7 +872,15 @@ def fetch_project(name, url, subdir, is_tarbz2=False):
           Ports.clear_project_build(name)
           return
 
-    fullpath = fullname + ('.tar.bz2' if is_tarbz2 else '.zip')
+    appended_extension = False
+    for _, extensions, _ in shutil.get_unpack_formats():
```

Comment:
I think this is python3 only... can we just hardcode a list of possible extensions?

Reply:
Done.